packages/icecream.rb

```diff
@@ -0,0 +1,137 @@
+# Adapted from Arch Linux icecream PKGBUILD at:
+# https://aur.archlinux.org/cgit/aur.git/plain/PKGBUILD?h=icecream
+
+require 'package'
+
+class Icecream < Package
+  description 'takes compile jobs from your build and distributes it to remote machines allowing a parallel build on several machines.'
```

Comment:
So is this like a torrent for builds?

Reply:
More like distcc with less configuration needed...